README.md

```diff
@@ -1,7 +1,11 @@
-Warning: This relies on an unreleased version of Dispatch.
-
 # Dispatch Catalog
 
-The official [Dispatch](https://docs.d2iq.com/ksphere/dispatch/latest/) catalog. Currently, this only hosts our Starlark standard library, however, this will grow to include other resources.
+![Image description](./images/dispatch-service-icon--medium.png)
+
+**Accelerate your CI/CD**
+
+D2iQ's [Dispatch](https://d2iq.com/solutions/ksphere/dispatch) helps organizations deliver CI/CD pipelines that support faster code development and deployment. The Dispatch catalog is a repository of libraries that Dispatch users can import into their pipelines.  Leverage the Dispatch catalog to accelerate common integration tasks such as testing, artifact management kubernetes resource usage and unit test execution. Bookmark or watch this page to leverage additional resources as they become available.
```

Comment:
> Bookmark or watch this page to leverage additional resources as they become available.

If we link back to the git repository, we can recommend they use Github's "watch" feature to be notified on releases.

Reply:
Actually this is the doc for the repo and not the site so this makes sense as-is.